libexec/rtld-elf/aarch64/rtld_start.S

```diff
@@ -330,7 +345,11 @@ ENTRY(_rtld_tlsdesc_dynamic)
 	stp	c13, c14, [csp, #(6 * 32)]
 	stp	c15, c16, [csp, #(7 * 32)]
 	stp	c17, c18, [csp, #(8 * 32)]
+#ifdef RTLD_SANDBOX
+	stp	c19,  c3, [csp, #(9 * 32)]
```

Comment:
This is OK because the original stack frame had an empty space for one capability. So I can store an extra capability without adjusting the size of the frame.

Reply:
Oh, right, these are `* 32` not `* 16`